src/index.js

```diff
@@ -5,6 +5,7 @@ const enter = document.querySelector('#location');
 fetch('https://api.giphy.com/v1/gifs/translate?api_key=DDO6wmRcjThYc6vRzRg6YFuHyCd0QOcR&s&s=Clouds', { mode: 'cors' })
   .then((response) => {
     if (!response.ok) {
+      alert(response.statusText);
```

Comment:
Good job trying to make the correction from the previous review, @mke2111 doesn't mean you should use `alert`, in fact, if you check the app example he shared with you, you will see that he didn't use `alert`.

Kindly implement this by using a pop-up div or modal.

![image](https://user-images.githubusercontent.com/31897434/121275973-b2465b80-c8c5-11eb-9240-0e26fa78335f.png)
[Pathwright](https://microverse.pathwright.com/library/javascript-module-v10-copy/166786/path/step/97635172/)